types/leaflet.vectorgrid/index.d.ts

```diff
@@ -0,0 +1,464 @@
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 3
+             */
+            tolerance?: number | undefined;
+
+            /**
+             * Tile extent (both width and height).
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 4096
+             */
+            extent?: number | undefined;
+
+            /**
+             * Tile buffer on each side.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 64
+             */
+            buffer?: number | undefined;
+
+            /**
+             * Logging level (0 to disable, 1 or 2).
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 0
+             */
+            debug?: number | undefined;
+
+            /**
+             * Whether to enable line metrics tracking for LineString/MultiLineString features.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default false
+             */
+            lineMetrics?: boolean | undefined;
+
+            /**
+             * Name of a feature property to promote to feature.id. Cannot be used with `generateId`.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default null
+             */
+            promoteId?: string | number | null | undefined;
+
+            /**
+             * Whether to generate feature ids. Cannot be used with `promoteId`.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default false
+             */
+            generateId?: boolean | undefined;
+
+            /**
+             * Max zoom in the initial tile index.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 5
+             */
+            indexMaxZoom?: number | undefined;
+
+            /**
+             * Max number of points per tile in the index.
+             *
+             * @external geojson-vt
+             * @see {@link https://github.com/mapbox/geojson-vt#options}
+             * @default 100000
+             */
+            indexMaxPoints?: number | undefined;
+        }
+
+        /**
+         * A `VectorGrid` for slicing up big GeoJSON or TopoJSON documents in vector
+         * tiles, leveraging [`geojson-vt`](https://github.com/mapbox/geojson-vt).
+         *
+         * @example
+         *
+         * ```
+         * var geoJsonDocument = {
+         * 	type: 'FeatureCollection',
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/leaflet.vectorgrid/index.d.ts;
  Line: 250;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.